### DISCUSSION TITLE
add readme in fold in inspector

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [Mariusthvdb/custom-more-info#5] [Ideas]
not yet found out how they do it but check this:

<img width="288" alt="Scherm­afbeelding 2024-01-01 om 18 05 21" src="https://github.com/Mariusthvdb/custom-more-info/assets/33354141/5de37dfb-1f2c-4233-980b-33864a69e0bd">

<img width="432" alt="Scherm­afbeelding 2024-01-01 om 18 05 29" src="https://github.com/Mariusthvdb/custom-more-info/assets/33354141/c3e8de47-c35e-462c-8489-e3a5799c90fa">

which is actually a nice little touch..

cool to implement on our custom-attributes/more-info ?

seems to be
```
console.groupCollapsed(`%cAUTO-ENTITIES ${Pt} IS INSTALLED`,"color: green; font-weight: bold"),console.log("Readme:","https://github.com/thomasloven/lovelace-auto-entities"),console.groupEnd());
```

**Comment by elchininet** [comment]
Yes, they use [console.groupCollapsed static method](https://developer.mozilla.org/en-US/docs/Web/API/console/groupcollapsed_static).
